### ISSUE TITLE
World whitelist not observed

### ISSUE THREAD

**Issue by SlimeDog** [PyroTempus/My-Plugins#46]
Spigot 1.15.1 and 1.15.2
PWP 3.5.2

I am experiencing acid water in world_blackdog, in which it should not occur. Configuration stipulates only two worlds.
```
Acidrain:
  Enabled: true
  DisabledInClaims: false
  Worlds:
    - 'world_whitedog'
    - 'world_yellowdog'
...
  Other:
    MakeWaterAcidic:
      Enabled: true
      AlwaysActive: true
```
If I disable PWP, there is no acid water (ie, no other plugin is producing it). What additional information can I provide?

**Comment by PyroTempus** [comment]
Fixed for v3.5.3.

**Comment by SlimeDog** [comment]
Confirmed fixed in PWP 3.5.3
